backend/src/routers/health.ts

```diff
@@ -0,0 +1,22 @@
+import express from 'express';
+import fs from 'fs';
+
+const version = (() => {
+    try {
+        return fs.readFileSync('./files/timestamp-commit').toString();
```

Comment:
What is this file, where is this updated?

Reply:
The plan is to modify the build-project npm script so that it assembles the required information and writes it to that file. This task is listed in Trello and will be done by Tommi soon.